examples/plot_custom_dataset_example.py

```diff
@@ -11,7 +11,7 @@
 
 import mne
 
-from braindecode.datasets.xy import create_from_X_y
+from braindecode.datautil.xy import create_from_X_y
```

Comment:
could you make it so that:

`from braindecode.datautil import create_from_X_y`

works and is the recommended way @sliwy ?